strings/station_name.txt

```diff
@@ -12,7 +12,7 @@ greek@=Alpha@,Beta@,Gamma@,Delta@,Epsilon@,Zeta@,Eta@,Theta@,Iota@,Kappa@,Lambda
 
 romanNum@=II@,III@,IV@,V@,VI@,VII@,VIII@,IX@,X@,XI@,XII@,XIII@,XIV@,XV@,XVI@,XVII@,XVIII@,XIX@,XX
 
-frontierLocations@=Typhon@,Plasma Star@,Fortitudo@,mAntistes@,Quadriga@,Fatuus@,Domus Dei@,Mundus@,Iustitia@,Iudicium@,Rota Fortuna@,Amantes@,Magus@,Regis@,Regina@,Mors@,Flaminica@,Senex@,Estella@,Shidd@,Abzu@,Fugg@,Frontier@,Channel@,Buttes@,Buttes Junction
+frontierLocations@=Typhon@,Plasma Star@,Fortitudo@,mAntistes@,Quadriga@,Fatuus@,Domus Dei@,Mundus@,Iustitia@,Iudicium@,Rota Fortuna@,Amantes@,Magus@,Regis@,Regina@,Mors@,Flaminica@,Senex@,Estella@,Shidd@,Šid@,Abzu@,Fugg@,Fugere@,Frontier@,Channel@,Buttes@,Buttes Junction
```

Comment:
could you verify this unicode works and doesn't break shit?

Reply:
![image](https://github.com/goonstation/goonstation/assets/51260013/ea5f3f17-b6aa-4a12-828b-b642b6113024)
Nothing looks broken to me.

Reply:
I mean in the window title and stuff when you actually rename it